README.md

```diff
@@ -26,6 +26,9 @@ Logo made by [Harshit Sharma](https://www.github.com/icoderharshit)
     - [Icon With No Background](#icon-with-no-background)
     - [Icon Positioned to The Left](#icon-positioned-to-the-left)
     - [Icon Positioned to The Right](#icon-positioned-to-the-right)
+  - [Toggle Button](#toggle-button)
+    -[Toggle Button as Checkboxes](#toggle-button-as-checkboxes)
```

Comment:
There's a problem in the indentation please fix it (make sure it's similar to the previous menu items)